### PR TITLE
Added sound files

### PR DESCRIPTION
**Pull request by betsythefc**
Added sound files for turrets

### CODE REVIEW

**Comment by reloZid** [comment]
Hmm, I merged a bit too early I think. I'm currently not able to play the files (tried with VLC). Any ideas?

**Comment by betsythefc** [comment]
Interesting... let me convert and try again. Ill confirm I can play them in VLC

On August 13, 2016 7:51:55 AM PDT, reloZid <notifications@github.com> wrote:
>Hmm, I merged a bit too early I think. I'm currently not able to play
>the files (tried with VLC). Any ideas?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/reloZid/android-anuto/pull/42#issuecomment-239624658

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by betsythefc** [comment]
I took a look and I am able to play the ones on my local machine, but not the ones I committed. Very interesting. Let me try again.

**Comment by betsythefc** [comment]
My mistake! I accidentally committed a bunch of shortcuts, not the actual files. I'm doing a new pull request
